### ISSUE TITLE
Release 1.5.0 does not pass unit tests

### ISSUE THREAD

**Issue by nim-nim** [optiopay/kafka#90]
```(shell)
+ go test -buildmode pie -compiler gc -ldflags ' -X github.com/optiopay/kafka/version=1.5.0 -extldflags '\''-Wl,-z,relro  '\'''
--- FAIL: TestTLSConnection (0.01s)
        connection_test.go:776: could not conect to test server: x509: certificate has expired or is not yet valid
FAIL
exit status 1
FAIL    github.com/optiopay/kafka       6.508s
```

Release unit tests should always work long term so they can be run when a security issue forces the update of the golang compiler and existing code needs checking for regressions.

(also there’s a typo in the error message)

**Comment by e-max** [comment]
You are right, thank you!  It was fixed in https://github.com/optiopay/kafka/pull/91

**Comment by dgellow** [comment]
Seems to be fixed, and v1.5.0 is now quite old. Closing.
